cms/apps.py

```diff
@@ -7,6 +7,7 @@ class CMSConfig(AppConfig):
     verbose_name = _("django CMS")
 
     def ready(self):
-        from cms.utils.setup import setup
+        from cms.utils.setup import setup, setup_cms_apps
 
         setup()
+        setup_cms_apps()
```

Comment:
Paulo is probably driving this but could the setup_cms_apps be in the original setup as it is part of that process.

Reply:
It could.... but it is a little easier to test this way :)